self-service-portal/src/api/UserService.js

```diff
@@ -28,9 +28,14 @@ export const serachUser = async (email) => {
     return post("/user/search", payload);
 }
 
-export const getAllUser = async () => {
+export const getAllUser = async (token="") => {
     var payload = { "filters": {} };
-    return post("/user/search", payload);
+    if(token != ""){
+        console.log("not blank ", token);
```

Comment:
avoid logging the user token. At least use a debug logger.

Reply:
debugging is not enabled in the app. ll take care of it later